crates/turborepo-repository/src/package_graph/mod.rs

```diff
@@ -341,15 +356,29 @@ impl PackageGraph {
                         != info.transitive_dependencies.as_ref()
                 })
                 .map(|(name, _info)| match name {
-                    WorkspaceName::Other(n) => Some(WorkspaceName::Other(n.to_owned())),
+                    WorkspaceName::Other(n) => {
+                        let w_name = WorkspaceName::Other(n.to_owned());
+                        Some(WorkspacePackage {
+                            name: w_name.clone(),
+                            path: self.workspace_dir(&w_name).unwrap().to_owned(),
```

Comment:
Can't we get this from the `info` field?
```suggestion
                            path: info.package_path().to_owned(),
```

Reply:
nice good catch, i didn't take your inline suggestion as it required a couple other changes. see new commit